ppl/zqd/handlers.go

```diff
@@ -132,7 +132,22 @@ func handleSearch(c *Core, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func handleWorker(c *Core, w http.ResponseWriter, httpReq *http.Request) {
+func handleWorkerRelease(c *Core, w http.ResponseWriter, httpReq *http.Request) {
+	w.WriteHeader(http.StatusNoContent)
+	if err := c.workerReg.RegisterWithRecruiter(httpReq.Context(), c.logger); err != nil {
+		// No point in responding with the error back to zqd root process,
+		// since this is happening on the cleanup after the search is finished.
+		c.logger.Warn(
+			"WorkerReleaseError",
+			zap.String("error", err.Error()),
+		)
+	}
+	c.logger.Info(
+		"WorkerReleaseWithRegister",
+	)
```

Comment:
I added this because I want "info" level logging of the release request. I will probably pull it out later after the load balancing and autoscaling systems are stable and trouble-free.

Reply:
removed now